build/Dockerfile.registry

```diff
@@ -1,10 +1,13 @@
-FROM quay.io/openshift/origin-operator-registry
+FROM quay.io/openshift/origin-operator-registry:4.2.0
 
-COPY manifests /registry
+COPY manifests manifests
+
+# Bundle should include only manifests related to the CSV,
+# drop all additional files.
+RUN find manifests/kubevirt-ssp-operator/*/ -type f ! -name '*_crd.*' ! -name '*.clusterserviceversion.*' -exec rm -f {} \;
```

Comment:
this works but I'm not too happy with it. We should log spurious/unexpected entries.

Reply:
maybe just use the `-v flag` of the `rm` command?